sdk/formrecognizer/azure-ai-formrecognizer/MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,630 @@
+
+New features provided by the `DocumentAnalysisClient` include having one consolidated function for analyzing general document layout, having a general prebuilt document model type, along with the same prebuilt model types that we included previously (receipts, invoices, business cards, identity documents), and custom models. Moreover, the models used in the newer version of the library, such as `AnalyzeResult`, remove many hierarchical dependencies between document elements and move them to a more top level and easily accesible position. The service has further improved how to define where elements are found on documents by moving towards `BoundingRegion` definitions where an element is returned with the bounding box per page where it is found. Document element fields are returned with more information, such as content and spans. 
+
+When using the `DocumentModelAdministrationClient` to build or compose new models users can now assign their own model ids and specify a description. Listing models on the administation client includes both prebuilt and custom models. When using `get_model()`, users can get the field schema for the model they specified, this includes prebuilt models. This client also provides functions for getting information from model operations.
+
+This table shows the relationship between SDK versions and supported API versions of the service
+
+|SDK version|Supported API version of service
+|-|-
+|3.2.0b1 - Latest beta release | 2.0, 2.1, 2021-09-30-preview
+|3.1.X - Latest GA release| 2.0, 2.1 (default)
+|3.0.0| 2.0
+
+Please refer to the [README](README) for more information on these new clients.
+
+## Important changes
+
+### Client usage
+
+We continue to support API key and AAD authentication methods when creating our clients. Below are the differences between the two versions:
+
+- In `3.2.x`, we have added `DocumentAnalysisClient` and `DocumentModelAdministrationClient` which support API version `2021-09-30-preview` and later.
+- `FormRecognizerClient` and `FormTrainingClient` will return an error if called with an API version of `2021-09-30-preview` or later. 
+- In `DocumentAnalysisClient` all prebuilt model methods along with custom model, layout, and a prebuilt document analysis model are unified into two methods called
+`begin_analyze_document` and `begin_analyze_document_from_url`.
+- In `FormRecognizerClient` there are two methods (a stream and URL method) for each of the prebuilt models supported by the service, this results in two methods for business card, receipt, identity document, and invoice models, along with a pair of methods for recognizing custom documents and for recognizing content/layout. 
+
+Creating a new clients in `3.1.x`:
```

Comment:
```suggestion
Creating new clients in `3.1.x`:
```

Reply:
I can do either:
`Creating a new client` or leave it as `Creating new clients`. Adding the a would make it singular. Let me know!

Reply:
plop. English. Creating new clients?

Reply:
Done!